src/DynamoCore/Graph/Notes/NoteModel.cs

```diff
@@ -79,5 +86,118 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
         }
 
         #endregion
+
+        #region BulletSupport
```

Comment:
Before reviewing this part can you add some comments to all of these privates as its not super clear what they are each doing or at least why they are doing it this way